2024-03-11 12:25:23 LogWrapper init() ./logs/error.log
2024-03-11 12:25:24 Bot started
2024-03-11 13:10:04 CRASH: 'dict' object has no attribute 'pipLocation'




